commands-yml/commands/device/recording-screen/start-recording-screen.yml

```diff
@@ -104,6 +104,7 @@ endpoint:
       type: string
       description: (iOS Only) The Frames Per Second rate of the recorded video.
                    Change this value if the resulting video is too slow or too fast. Defaults to 10.
+                   This can decrease the resulting file size so that it will skip more frames drecreasing the value.
```

Comment:
I don't think `"so that it will skip more frames drecreasing the value"` is needed. Just
```
This can decrease the resulting file size.
```